2021-03-15 18:53:56,283 INFO    MainThread:8689 [internal.py:wandb_internal():88] W&B internal server running at pid: 8689, started at: 2021-03-15 18:53:56.283236
2021-03-15 18:53:56,284 DEBUG   HandlerThread:8689 [handler.py:handle_request():101] handle_request: run_start
2021-03-15 18:53:56,285 INFO    WriterThread:8689 [datastore.py:open_for_write():77] open: /Users/truongtn/Desktop/Desktop/HocTap/Semester8/Python/YoloV5/yolov5/wandb/offline-run-20210315_185355-287tg3zr/run-287tg3zr.wandb
2021-03-15 18:53:57,904 DEBUG   HandlerThread:8689 [meta.py:__init__():34] meta init
2021-03-15 18:53:57,904 DEBUG   HandlerThread:8689 [meta.py:__init__():48] meta init done
2021-03-15 18:53:57,904 DEBUG   HandlerThread:8689 [meta.py:probe():190] probe
2021-03-15 18:53:57,905 DEBUG   HandlerThread:8689 [meta.py:_setup_git():180] setup git
2021-03-15 18:53:57,923 DEBUG   HandlerThread:8689 [meta.py:_setup_git():187] setup git done
2021-03-15 18:53:57,923 DEBUG   HandlerThread:8689 [meta.py:_save_pip():52] save pip
2021-03-15 18:53:57,923 DEBUG   HandlerThread:8689 [meta.py:_save_pip():66] save pip done
2021-03-15 18:53:57,923 DEBUG   HandlerThread:8689 [meta.py:probe():231] probe done
2021-03-15 18:59:01,800 WARNING MainThread:8689 [internal.py:is_dead():343] Internal process exiting, parent pid 8658 disappeared
2021-03-15 18:59:01,808 ERROR   MainThread:8689 [internal.py:wandb_internal():142] Internal process shutdown.
2021-03-15 18:59:01,860 INFO    SenderThread:8689 [sender.py:finish():872] shutting down sender
2021-03-15 18:59:01,871 INFO    HandlerThread:8689 [handler.py:finish():535] shutting down handler
2021-03-15 18:59:01,899 INFO    WriterThread:8689 [datastore.py:close():258] close: /Users/truongtn/Desktop/Desktop/HocTap/Semester8/Python/YoloV5/yolov5/wandb/offline-run-20210315_185355-287tg3zr/run-287tg3zr.wandb
2021-03-15 18:59:01,911 INFO    MainThread:8689 [internal.py:handle_exit():78] Internal process exited
